ui/src/Pages/PeoplePage/SingleFaceGroup/MergeFaceGroupsModal.tsx

```
@@ -59,59 +66,194 @@ const MergeFaceGroupsModal = ({
+      open: isOpen,
+    },
+    searchTitle:
+      t(
+        'people_page.modal.merge_face_groups.sources_table.title',
+        'Select one or more source faces to merge into:'
+      ) +
+      ` ${
+        selectedDestinationFaceGroup?.label ??
+        t('people_page.face_group.unlabeled', 'Unlabeled') ??
+        'Unlabeled'
+      }`,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)